cmd/corald/routes/routes.go

```diff
@@ -112,6 +112,12 @@ func routes(a *app.App) {
 				return "/v1/exec/" + c.Params["query_set"]
 			}))
 
+	// Execute xenia queries directly.
+	a.Handle("GET", "/v1/exec/:query_set", handlers.Proxy(xeniadURL, nil))
```

Comment:
@wyatt having the actual URL's so far away from this code does not help with readability. Especially if the url is only used here. I feel like we don't need the constant and just place the url in this call.